compute_tools/src/compute.rs

```diff
@@ -1675,10 +1672,26 @@ impl ComputeNode {
 
         let unknown_op = "unknown".to_string();
         let op_id = spec.operation_uuid.as_ref().unwrap_or(&unknown_op);
-        info!(
-            "finished reconfiguration of compute node for operation {}",
-            op_id
-        );
+        info!("finished reconfiguration of compute node for operation {op_id}");
+
+        Ok(())
+    }
+
+    /// Tell postgres/pgbouncer/local_proxy to reload their configurations.
+    #[instrument(skip_all)]
+    pub fn reload<'a>(&self, spec: ComputeSpec) -> Result<()> {
```

Comment:
It doesn't seem like the lifetime is doing anything. Maybe I am wrong.

Reply:
Yeah, was from a prev iteration of the code